Dockerfile

```diff
@@ -1,5 +1,37 @@
-FROM ubuntu:bionic
+FROM debian:stable-slim as build
 WORKDIR /usr/src
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt update -y
-RUN apt install -y curl cmake libgl1-mesa-dev libsdl2-dev libcurl4-openssl-dev libopenal-dev libfreetype6-dev mingw-w64 g++-mingw-w64 g++-multilib git zip vim-nox rsync
+RUN apt install -y make gcc g++ libgl1-mesa-dev libsdl2-dev libcurl4-openssl-dev libopenal-dev libfreetype6-dev zip rsync
+COPY . /usr/src/
+RUN make -j
+
+FROM debian:stable-slim as run
```

Comment:
Have you pushed a copy of this new layer to dockerhub that I can try?

Reply:
https://hub.docker.com/repository/docker/erentar2002/tremulous

Reply:
Can i reach you on irc/discord/elsewhere?